lib/generate-index.js

```diff
@@ -1,6 +1,28 @@
 'use strict'
 
 const lunr = require('lunr')
+require("lunr-languages/lunr.stemmer.support")(lunr)
```

Comment:
We should probably require/load them on demand no? If `DOCSEARCH_LANGS` is not defined there's no extra work needed.

Reply:
Yes indeed.